src/Server.ts

```diff
@@ -1,15 +1,18 @@
-import { IConfig } from './config/IConfig';
+import { IConfig, notFoundRoute, errorHandler } from './index';
```

Comment:
- IConfig file should be imported from config folder.
- notFoundRoute, errorHandler should be imported from libs folder